Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift

```diff
@@ -674,7 +674,7 @@ class ChannelListViewController :
 
         let src_vc = segue.source as! UserDetailViewController
         
-        let vc = self.storyboard?.instantiateViewController(withIdentifier: "Text Message") as! TextMessageViewController
+        let vc = self.storyboard?.instantiateViewController(withIdentifier: format("Message - %@", getDisplayName(src_vc.userid)) as! TextMessageViewController
```

Comment:
"Text Message" is the ID of a View Controller in the Story Board.
Can you remove this last commit? The commits before looks ok.